requirements/chart-tests.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with python 3.9
+# This file is autogenerated by pip-compile with python 3.8
```

Comment:
Oh my bad, will fix tomorrow, the venv created was using python 3.8, I'll pip-compile it again with python 3.9